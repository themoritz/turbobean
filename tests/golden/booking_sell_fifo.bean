2020-01-01 open Assets:Stocks AAPL "FIFO"
2020-01-01 open Assets:Cash

2020-01-02 * "Buy"
  Assets:Stocks   10 AAPL @ 10 USD
  Assets:Cash   -100 USD

2020-01-03 txn "Buy"
  Assets:Stocks   10 AAPL @ 15 USD
  Assets:Cash

2020-01-04 txn "Sell"
  Assets:Stocks  -15 AAPL @ 30 USD
  Assets:Cash    450 USD
