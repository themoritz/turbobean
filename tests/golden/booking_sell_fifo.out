Assets     5 AAPL @ 15 USD {2020-01-03}
           200 USD
├ Stocks   5 AAPL @ 15 USD {2020-01-03}
└ Cash     200 USD
